hydra.yaml

```
@@ -0,0 +1,56 @@
+package:
+  name: hydra
+  version: 2.3.0
+  epoch: 0
+  description: OpenID Certified OAuth 2.0 Server and OpenID Connect Provider
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1" # Needed for SQLite support
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ory/hydra
+      tag: v${{package.version}}
+      expected-commit: ee8c339ddada3a42529c0416897abc32bad03bbb
+
+  - uses: go/install
+    with:
+      package: .
+      ldflags: -X 'github.com/ory/hydra/v2/driver/config.Version=$(git describe --abbrev=0 --tags)' -X 'github.com/ory/hydra/v2/driver/config.Date=$(TZ=UTC date -u "+%Y-%m-%dT%H:%M:%SZ")' -X 'github.com/ory/hydra/v2/driver/config.Commit=$(git rev-parse HEAD)'
+      tags: sqlite,json1,hsm
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}/usr/local/bin"
+      cp /home/build/melange-out/${{package.name}}/usr/bin/hydra "${{targets.destdir}}"/usr/local/bin/hydra
```

I believe by default we are in the working directory of `/home/build/melange-out/${{package.name}}` so we don't need to add it again, we should do probably something like this:

```suggestion
      cp /usr/bin/hydra "${{targets.destdir}}"/usr/bin/hydra
```

I don't think we should have `local` here too as I don't see them in upstream https://github.com/ory/hydra/blob/96d4ffe2cf3c6fce40d968f343088be39f8de165/.docker/Dockerfile-alpine#L17, any reference from where you got it?

---

I remember picking it from somewhere, let me find it

---

cant recall now :(
I have removed that block (default in bin), updated tests and pushed the commit 🤞